packages/client/src/declarations/submissionMiddleware.test.ts

```diff
@@ -143,26 +143,31 @@ describe('Submission middleware', () => {
         mockDeclarationData.registration.certificates[0] = {
           collector: {
             relationship: 'OTHER',
-            affidavit: {
+            affidavitFile: {
               contentType: 'image/jpg',
               data: 'data:image/png;base64,2324256'
             },
-            individual: {
-              name: [{ firstNames: 'Doe', familyName: 'Jane', use: 'en' }],
-              identifier: [{ id: '123456', type: 'PASSPORT' }]
-            }
+            name: [{ firstNames: 'Doe', familyName: 'Jane', use: 'en' }],
+            identifier: [{ id: '123456', type: 'PASSPORT' }]
           },
           hasShowedVerifiedDocument: true,
-          payments: [
-            {
-              paymentId: '1234',
-              type: 'MANUAL',
-              amount: 50,
-              outcome: 'COMPLETED',
-              date: '2018-10-22'
-            }
-          ],
-          data: 'data:image/png;base64,2324256'
+          certificateTemplateId: 'certified-birth-certificate',
+          templateConfig: {
```

Comment:
Is this still waiting for an update? I don't think we need to store the template configuration anymore as we can always get it from the list of certificates with the id right?

Reply:
Yeah, I have been thinking about it too, we should get the templateConfig only when we need it, it will simplify a lot of things. Will resolve this conversation once it is implemented.